spec/spec_helper.rb

```diff
@@ -61,31 +60,75 @@
       SpecGlobalLogger.logger.debug  { "stopping zookeeper service" }
       ZK::Server.shutdown
     end
+
+    config.before(:each, zookeeper: true) do
```

Comment:
You don't need this whole change if you aren't testing throttling and lost connections. I accidentally left these changes in here. I used them in my gem https://github.com/nerdrew/ruby-zk_recipes to test lost connections and degraded connections.

Reply:
agreed. could maybe your extensions be worth merging to zk? I'm a fan of having fewer, better maintained gems, with more maintainers in general :)

Reply:
I'm happy to merge my cache recipe into this gem if the maintainers are cool with it. That was my original intention, but got no responses on the issues I opened.

Reply:
In fact, I'd like to make some changes to logging in this gem. With lots of watches, a ZK restart took our app down b/c of the ZK gem logging volume.

Reply:
Hey, so, to give you a bit of the backstory, @eric and I worked on this together back in like 2011 or thereabouts. I was using it for a task queuing system and he was using it for his company. I took the original zookeeper repo over from Twitter, and added zk on top of it.

I haven't _really_ touched the code since 2013 (though it's still in active use in parts of twitter). @eric was keeping on top of things because he was still using it actively, until he sold his company a year ago and has been kinda just enjoying life.

Basically what I'm saying is, I haven't been an active user of this codebase for a long time. Hell, I haven't written any *ruby* in over 4 years. If you guys are using zk in production, and are willing to put in the work to do releases and maintenance of this, then by all means, I'm happy to help out however I can (including if that means getting out of the way).

Reply:
Want to add some people to the contributor list so we can review PRs, release versions, etc?

Reply:
I'd like to merge my cache recipe (https://github.com/nerdrew/ruby-zk_recipes) into this gem  but I'm worried about not being able to make changes quickly.